client/src/modules/cohort/components/CohortHeader.jsx

```diff
@@ -188,7 +196,7 @@ class CohortHeader extends PureComponent {
           />
         ) : (
           <Fragment>
-            {description && (
+            {description.trim() && (
```

Comment:
maybe use teranry operator here:  description.trim() ? (<p><p/>) : (<button></button>)

Reply:
https://eslint.org/docs/rules/no-nested-ternary

it would be nesting ternary inside ternary.